mtags/src/main/scala/scala/meta/internal/mtags/CommonMtagsEnrichments.scala

```diff
@@ -355,7 +355,7 @@ trait CommonMtagsEnrichments {
     def isAmmoniteGeneratedFile: Boolean =
       doc.endsWith(".amm.sc.scala")
     def isAmmoniteScript: Boolean =
-      isScalaScript && !isWorksheet
+      isScalaScript && !isWorksheet && !doc.endsWith("build.sc")
```

Comment:
We should accept scripts like `test-build.sc` etc.
```suggestion
      isScalaScript && !isWorksheet && !doc.endsWith("/build.sc")
```